Dockerfile

```
@@ -1,12 +1,12 @@
 ############# builder
-FROM golang:1.15.5 AS builder
+FROM eu.gcr.io/gardener-project/3rd/golang:1.15.5 AS builder
 
 WORKDIR /go/src/github.com/gardener/gardener-extension-shoot-cert-service
 COPY . .
 RUN make install
 
 ############# gardener-extension-shoot-cert-service
-FROM alpine:3.12.0 AS gardener-extension-shoot-cert-service
+FROM alpine:3.12.1 AS gardener-extension-shoot-cert-service
```

You should want also
```suggestion
FROM eu.gcr.io/gardener-project/3rd/alpine:3.12.1 AS gardener-extension-shoot-cert-service
```